main.js

```diff
@@ -1,104 +1,68 @@
 let articles;
 let resultsPage;
 
+const sortControls = document.getElementById('sort-by-container');
 const sortSelect = document.getElementById('sort-by-select');
-const filtersButton = document.getElementById('filters-button');
 const filterMenu = document.getElementById('filters-container');
+const filtersButton = document.getElementById('filters-button');
 const queryInput = document.getElementById('query-input');
-const previousPageButton = document.getElementById('previous-page-button');
-const nextPageButton = document.getElementById('next-page-button');
+const searchResultsDiv = document.getElementById('search-results-container');
 const articlesContainer = document.getElementById('articles-container');
+const pageBottom = document.getElementById('page-bottom');
 
 bindEvents();
 
 function bindEvents() {
+  window.addEventListener('load', () => {
```

Comment:
ah, can an intersection observer only be added during load?

Reply:
No, this isn't necessary. We can safely set up the observer directly without having to listen for any events first. I'm leaning towards still adding the observer inside of `bindEvents()` though because it seems symantically similar to adding an event listener. What do you think?